docs/apache-airflow-providers-http/operators.rst

```diff
@@ -118,3 +118,24 @@ Here we pass form data to a ``POST`` operation which is equal to a usual form su
     :language: python
     :start-after: [START howto_operator_http_task_post_op_formenc]
     :end-before: [END howto_operator_http_task_post_op_formenc]
+
+.. _howto/operator:PaginatedHttpOperator:
+
+You can also do a series of call on a paginated API:
+
+The :class:`~airflow.providers.http.operators.paginated.SimpleHttpOperator` allows to repeatedly call an API
```

Comment:
The documentation does not match to current code anymore. Is this intended?

Reply:
Indeed, docs was still mentioning "PaginatedHttpOperator". I removed all mentions of it. Thanks !